crates/bevy_text/src/text.rs

```diff
@@ -216,9 +209,10 @@ impl From<JustifyText> for cosmic_text::Align {
     }
 }
 
-#[derive(Clone, Debug, Reflect)]
-/// `TextStyle` determines the style of the text in a section, specifically
+/// `TextStyle` determines the style of a text span within a [`TextBlock`], specifically
 /// the font face, the font size, and the color.
+#[derive(Component, Clone, Debug, Reflect)]
+#[reflect(Component, Default, Debug)]
 pub struct TextStyle {
```

Comment:
The color should be in a separate component `TextColor`, so that changes to the color don't trigger change detection and a relayout of the text.

Reply:
Maybe `TextStyle` should be renamed as well to `TextFont` or something. It would be consistant maybe with the UI `Style` component if `TextBlock` was renamed to `TextStyle`, but `Style` should be renamed too.

Reply:
This is fine with me but can we do it in a separate PR? This one is already quite huge. The migration from TextStyle to TextFont/TextColor will by itself be a big chore and I call 'not it'.

Reply:
Yup thats fine by me.